basics/Strings examples/character count without methods(using for).py

```diff
@@ -0,0 +1,9 @@
+str = input("Type the string:")

```

Comment:
This overwrites the `str` builtin. See [here](https://github.com/Akuli/python-tutorial/blob/master/basics/using-functions.md#variables-names-and-built-in-things) for an explanation. Please use some other variable name than `str`.

Also, `ch` and `i` are too short variable names. Make them more descriptive (without making them annoyingly long).

Reply:
sorry..
Forgot about that..
I will change it..